Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OneSignal",
+    products: [
+        .library(
+            name: "OneSignal",
+            type: .static,
+            targets: ["OneSignal"]),
+        .library(
+            name: "OneSignalDynamic",
+            type: .dynamic,
+            targets: ["OneSignal"]),
+    ],
+    targets: [
+        .target(
+            name: "OneSignal",
+            dependencies: [],
+            path: "iOS_SDK/OneSignalSDK/",
+            sources: ["Source"],
+            publicHeadersPath:"Source"),
```

Comment:
Using "Source" for `publicHeadersPath` results in all internal classes being accessible. Can we scope this down so only `OneSignal.h` is accessible?

Some ideas on how this could be done is pointing to one of the following paths if a specific file isn't possible:
* `iOS_SDK/OneSignalSDK/Framework/OneSignal.framework/Headers`
* `iOS_SDK/OneSignalSDK/Framework/OneSignal.framework/Versions/A/Headers`

Reply:
fixed this!

Reply:
A new folder was made: `SwiftPM/Public/Headers`
https://github.com/OneSignal/OneSignal-iOS-SDK/pull/712/files#diff-37ca2dd15ca0f6b1b49e78db084ef5b9R19